index.bs

```diff
@@ -5486,6 +5488,11 @@ Changes since earlier versions</h3>
 		"href": "https://www.w3.org/Guide/transitions",
 		"title": "Organize a Technical Report Transition",
 		"publisher": "W3C"
+	},
+	"HREVIEW": {
+		"href": "https://www.w3.org/Guide/documentreview/",
+		"title": "How to do Wide Review",
```

Comment:
It's super confusing to have a term definition of "Horizontal Review" that points to a document about Wide Review. See also w3c/documentreview#12. Any reason why not to extend the change in this pull request from Horizontal Review to Wide Review?

Reply:
I think the issue raised in https://github.com/w3c/documentreview/issues/12 is right, and that the title of that document is wrong. It claims to be about wide review, but really is about horizontal review. For this change, we need a reference to a document that explains how horizontal review is done, and that document is it, even if it has a weird title.

Reply:
For the record, I disagree regarding w3c/documentreview#12. In my view there should be a document about wide review, and be clear in that document about the subset of wide review that is horizontal review. Conflating them or forgetting about the "inconvenient" part of wide review that is not horizontal review is, in my view, a mistake.

Reply:
Given that that document is still a moving target with disagreements about what it ought to become, I'm dropping that reference for now, and keeping the rest as is.